### PR TITLE
Fixes ReferenceError with webapp dev container.

### PR DESCRIPTION
**Pull request by robotoer**
Hey Kathryn,

That error you emailed me about was caused by a faulty line of code when the development docker container specifically is used. The quick version:

I added a file called `runtime.js` that gets bypassed by webpack's bundling. This file is overwritten by the `entrypoint.sh` script in the production container. This script injects environment variables into the `runtime.js` file that are prefixed with `REACT_APP_` since the production container has an already built/bundled copy of the flow webapp (thus the react-scripts and friends can't inject environment variables into the `process` global). This is not necessary in the development container of course (we run `npm start` there to generate a built/bundled copy on source-code edit/save).

You might wonder why I didn't just have the `entrypoint.sh` script write to the `process` global instead of writing to a new `runtime` global. Unfortunately it seems that the `process` global gets overwritten during react startup meaning we must use a separate global.

The helper function that I wrote to fetch environment variables (by first checking if they're in `runtime` and then if they're in `process.env`) was using the following condition: `if (runtime && ...`. Changing this to `if (window.runtime && ...` prevents a crash due to an unreferenced global when we're trying to check to see if it is in fact defined or not.

*TL;DR*
Merge this PR in to fix your local development environment

### CODE REVIEW

**Review by kathryn-explorable — APPROVED** [review]
Confirming that I tested and this works!
Merging.
